message/pool.go

```diff
@@ -34,6 +35,8 @@ type Pool struct {
 	validator     PoolValidator
 	pending       map[cid.Cid]*timedmessage // all pending messages
 	addressNonces map[addressNonce]bool     // set of address nonce pairs used to efficiently validate duplicate nonces
+
+	journal journal.Journal
```

Comment:
For now, don't journal this. It's massively higher traffic than the outbox and the need is less clear. I think we should do the command API before this, but in any case let's defer that discussion to new PRs.